src/scenes/App/components/Loader/index.js

```diff
@@ -37,12 +36,9 @@ export default class Loader extends Component {
             <img className={classes.loaderGif} src="/images/loader.gif" alt="Loading..." />
           </div>
           <div className={classes.loaderPercentWrapper}>
-            <Typography variant="display1" className={classes.loaderPercent}>{syncPercent}</Typography>%
-            <p>
-              <FormattedMessage id="str.blockSync" defaultMessage="Blockchain syncing..." />
-              &nbsp;
-              <FormattedMessage id="str.wait" defaultMessage="Please wait." />
-            </p>
+            <Typography variant="display1" className={classes.loaderPercent}>{syncPercent}</Typography>
+            <p>%</p>
```

Comment:
keep `%` in the same line with `syncPercent`

Reply:
why, <p></p> is a different component, and they will still be on the same line.